superset/utils/hashing.py

```diff
@@ -21,7 +21,7 @@
 
 
 def md5_sha_from_str(val: str) -> str:
-    return hashlib.md5(val.encode("utf-8")).hexdigest()
+    return hashlib.md5(val.encode("utf-8")).hexdigest()  # noqa: S324
```

Comment:
Seems this would break a bunch of calls as md5_sha_from_str is used throughout the codebase (few dozen calls). Also I think this hash is for internal use, not to secure things (?) Might also break things if we assume a deterministic hash anywhere...

Reply:
Let's label this as "out of scope for this PR"